style.css

```diff
@@ -1,65 +1,54 @@
 body {
-    padding: 0;
-    margin: 0;
-    font-family: Arial, Helvetica, sans-serif;
-    background-color: rgb(226, 226, 226);
-}
-
-header {
-    text-align: center;
-}
-
-div.todo {
-    padding: 1em;
-}
+  padding: 0;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #e2e2e2; }
 
 .container {
-    max-width: 25em;
-    margin: 0 auto;
-    background-color: white;
-    box-shadow: 0 0 10px rgb(172, 172, 172);
-}
-
-/* New Todo Field */
-#new-todo {
+  max-width: 25em;
+  margin: 0 auto;
+  background-color: white;
+  box-shadow: 0 0 10px #acacac; }
+
+.todo {
+  padding: 1em;
+  /* New Todo Entry Field */ }
+  .todo .header {
+    text-align: center; }
+  .todo .new-todo {
+    display: block;
     box-sizing: border-box;
     width: 100%;
     font-size: 1em;
-    padding: .5em;
-}
-
-/* Mark all as completed field */
-.small {
-    font-size: .75em;
-}
+    padding: .5em; }
+  .todo .checkbox-field--small {
+    font-size: .75em; }
+  .todo .todo-items {
+    margin: 1em 0; }
+
+.todo-items ul {
+  list-style: none;
+  padding: 0;
+  margin: 0; }
+
+.todo-items ul li {
```

Comment:
Avoid selecting DOM elements like this. These should have their own CSS class to target instead.